models/equaler.go

```diff
@@ -5,4 +5,22 @@ type Equaler interface {
 	// Returns true if the Equaler object is the same as this object; otherwise false is returned.
 	// You need to convert the Equaler object using type assertions: https://golang.org/ref/spec#Type_assertions
 	Equal(Equaler) bool
-}
\ No newline at end of file
+}
+
+// DummyEqualer implements the Equaler interface and can be used by tests.
+// Other than that it has not meaning.
+type DummyEqualer struct {
```

Comment:
Seems it's only used for testing, if so, could we move it to one of the "_test.go" files?